### PR TITLE
Googlemap page

### PR DESCRIPTION
**Pull request by AndresEstebanGonzalez**
Second page (Googlemaps background) is near complete. Need to go back and see the font sizes. Need to put navbar to fixed and turn black.

### CODE REVIEW

File: styles/_googlemaps.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.googlemaps {
+  width: 100vw;
+  height: 100vh;
+  padding: 0 1rem 0 0;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: This case is better to use `padding-right: 1rem`

File: styles/_googlemaps.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  iframe {
+    width: 100vw;
+    height: 100vh;
+    border: 0 none;
+    padding: 0 1rem 0 0;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: Same here

File: styles/_googlemaps.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    width: 34rem;
+    height: 100vh;
+    padding: 1rem 1rem 1rem 2rem;
+    background-color: rgba(0, 0, 0, 0.8);
+  }
+  .right--sector-text {
+    margin-top: 4rem;
+    text-align: right;
+  }
+  .right--sector-text h3 {
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: not sassy enough

```
.right--sector-text {
...
h3 {
...stuff
}
}
```

File: styles/_googlemaps.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    color: #dddddd;
+    font-size: 24px;
+    font-weight: 400;
+    text-align: right;
+    strong {
+      color: #6ab04c;
+      font-weight: 700;
+    }
+  }
+  .right--sector-text p {
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: same as above

File: styles/styles.scss [hunk 5]
```diff
@@ -1,5 +1,7 @@
 @import url("https://fonts.googleapis.com/css?family=Roboto:400,500,900&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
```
Comment (nathan-hyan) [inline styles/styles.scss]: Google fonts lets you import various fontweights into one link

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Some comments

File: styles/_googlemaps.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
+.googlemaps {
+  width: 100vw;
+  height: 100vh;
+  padding-right: 1rem;
+  iframe {
+    width: 100vw;
+    height: 100vh;
+    border: 0 none;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: `border: none;` will do the trick just fine

File: styles/_googlemaps.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  .googlemaps--container {
+    width: 34rem;
+    height: 100vh;
+    float: right;
+
+    .googlemaps--container-right-sector {
+      width: 34rem;
+      height: 100vh;
+      padding: 1rem 1rem 1rem 2rem;
+      background-color: rgba(0, 0, 0, 0.8);
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: use variables for colors

File: styles/_googlemaps.scss [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    .right--sector-text {
+      margin-top: 4rem;
+      text-align: right;
+      h3 {
+        color: #dddddd;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: same here

File: styles/_googlemaps.scss [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        font-size: 24px;
+        font-weight: 400;
+        text-align: right;
+        strong {
+          color: #6ab04c;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: same here

File: styles/_googlemaps.scss [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+        font-weight: 300;
+      }
+    }
+
+    .googlemaps--link-div {
+      margin-top: 3rem;
+    }
+    #googlemaps--link {
+      font-size: 20px;
+      color: #6ab04c;
```
Comment (nathan-hyan) [inline styles/_googlemaps.scss]: same here

File: styles/_navbar.scss [hunk 11]
```diff
@@ -36,7 +36,7 @@
       list-style: none;
     }
     .navbar-list a {
-      color: #dddddd;
+      color: white;
```
Comment (nathan-hyan) [inline styles/_navbar.scss]: use variables with colors

**Review by AndresEstebanGonzalez — COMMENTED** [review]
Comments review and corrected.

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
More comments

File: styles/_colors.scss [hunk 12]
```diff
@@ -0,0 +1,5 @@
+$color-black: #000000cc;
+$color-green: #6ab04c;
+$color-white-grey: #dddddd;
+$color-white: white;
```
Comment (nathan-hyan) [inline styles/_colors.scss]: use only hex values

File: styles/_colors.scss [hunk 13]
```diff
@@ -0,0 +1,5 @@
+$color-black: #000000cc;
```
Comment (nathan-hyan) [inline styles/_colors.scss]: Hex values can only have either 3 or 6 numbers

File: styles/_colors.scss [hunk 14]
```diff
@@ -0,0 +1,5 @@
...
+$color-green: #6ab04c;
+$color-white-grey: #dddddd;
+$color-white: white;
+$color-black-transparent: #000000ea;
```
Comment (nathan-hyan) [inline styles/_colors.scss]: in this case AND ONLY WITH TRANSPARENCY you can use rgba()

`rgba($black, 0.5)`

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Last comments

**Review by nathan-hyan — APPROVED** [review]
🥇

**Pull request merged**